core/src/commonMain/kotlin/com/xebia/functional/xef/llm/models/chat/Role.kt

```diff
@@ -0,0 +1,8 @@
+package com.xebia.functional.xef.llm.models.chat
+
+enum class Role {
+  system,
+  user,
+  assistant,
+  function
```

Comment:
```suggestion
  SYSTEM,
  USER,
  ASSISTANT,
  FUNCTION
```

We're using `ALLCAPS` and `AllCaps` notation, for `enum class`. It seems this is already inconsistent (we should fix that), the recommended style is `ALLCAPS` or `ALL_CAPS` (streaming snake case).

https://kotlinlang.org/docs/coding-conventions.html#property-names

Reply:
Adressed in https://github.com/xebia-functional/xef/pull/196/commits/ef1c24cac6bd635ff2fb8c08802fd75cdbf26697